timelock-agent/src/main/java/com/palantir/timelock/paxos/TimeLockAgent.java

```diff
@@ -174,17 +174,16 @@ private void createAndRegisterResources() {
         // Finally, register the health check, and endpoints associated with the clients.
         TimeLockResource resource = TimeLockResource.create(namespaces);
         healthCheck = paxosResources.leadershipComponents().healthCheck(namespaces::getActiveClients);
-        LockWatchTestingService.create(
-                Suppliers.compose(TimeLockRuntimeConfiguration::lockWatchTestConfig, runtime::get),
-                resource::getLockWatchingResource);
+
         registrar.accept(resource);
 
-        Function<String, AsyncTimelockService> serviceCreator =
-                namespace -> namespaces.get(namespace).getTimelockService();
+        Function<String, AsyncTimelockService> creator = namespace -> namespaces.get(namespace).getTimelockService();
         if (undertowRegistrar.isPresent()) {
-            undertowRegistrar.get().accept(ConjureTimelockResource.undertow(redirectRetryTargeter(), serviceCreator));
+            undertowRegistrar.get().accept(ConjureTimelockResource.undertow(redirectRetryTargeter(), creator));
```

Comment:
sad about this :( but I can't seem to think of a way without having some sort of conjure-undertow generation support for handling exceptions or having a proxy etc.

Reply:
😢